bin/list

```diff
@@ -161,6 +161,34 @@ const renderDirectory = async dir => {
   return render(details)
 }
 
+
+
+// To enhance, one can take inspiration from https://github.com/jfhbrook/node-ecstatic/blob/fe17a5623c18319f82101b6549e9e2e526e03d1f/lib/ecstatic.js#L284
+const isNotModified = (req, etag, fileStats) => {
+  try {
```

Comment:
@fearphage I don't know what can throw now, or what can throw in the future (if someone modifies the code), but I'm sure that failsafe behavior (caching disabled) is probably better than total unability to serve files